src/efs_parser/BigWig.py

```diff
@@ -184,7 +184,15 @@ def getRange(self, chr, start, end, bins=2000, zoomlvl=-1, metric="AVG", respTyp
             if respType == "DataFrame":
                 result = toDataFrame(values, self.columns)
                 result["chr"] = chr
+                # checking range by: Hany Elgaml
 
+                query_string = '(start >= ' + str(start) + ' and start <= ' + str(end) + ') or (end >= ' + str(
+                    start) + ' and end <= ' + str(end) + ')'
```

Comment:
I think we can simplify this ever further 

```
query_string = f"start <= {end} and end >= {start}" 
```

also checkout f strings in python

Reply:
Done